### PR TITLE
Improve Tealium support

### PR DESCRIPTION
**Pull request by muodov**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206215861735514

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -ex
 
 ESBUILD="node_modules/.bin/esbuild --bundle"
 
-$ESBUILD --format=iife --target=chrome90 --minify playwright/content.ts --outfile=dist/autoconsent.playwright.js
+$ESBUILD --format=iife --target=es2021 --minify playwright/content.ts --outfile=dist/autoconsent.playwright.js
```
Comment (sammacbeth) [inline build.sh:6]: Why do we need to change this? Are we missing some syntax support that's needed? It looks to be a safe change, as this is just for playwright, but perhaps we can split it into a separate PR?

**Review by sammacbeth — COMMENTED** [review]
Looks good, I'm just wondering if the build changes should be in this PR or split out.

File: build.sh (hunk 1 above)
Comment (muodov) [reply build.sh:6]: This is needed because otherwise optional chaining is transpiled weirdly and the new eval snippets don't work in the tests

**Review by sammacbeth — APPROVED** [review]
